duckduckgo/iOS
# os_log fixes

Task/Issue URL: https://app.asana.com/0/414235014887631/1205157723964854/f
BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/437

**Description**:
- fixes "implicit import" warning
- fixes "<private>" message in Console app for interpolated messages
- outputs .debug messages to Console app for DEBUG builds



**Steps to test this PR**:
1. Validate macOS & iOS projects build&run, there shouldn‘t be "implicit import" warning for the .disabled OSLog default arguments
2. add interpolated os_log output, validate it‘s shown in Console app: `os_log("some interpolated \(value)")`
3. validate `.debug` level os_log is shown in Console app: `os_log("test debug message", type: .debug)`
